### PR TITLE
Img updates

### PR DESCRIPTION
**Pull request by francalovescakes**
I've updated the following:
- new network diagrams that now show the direction of traffic Eg NATGW is egress-only
- a couple notes about NATGW
- updated AWS UI screenshots thanks to Lachy
- added new LB type into docs and a link to their comparisons
- .gitignore so .DS_Store files aren't committed

### CODE REVIEW

File: part-2/2.2_setting_up_your_load_balancer.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 - ALB - HTTP/S (layer 7)
 - NLB - TCP, UDP (layer 4)
-- ELB - Basic load balancing for EC2 👈🏿 This is what we're going to work with!
+- GLB - To manage a fleet of third-party virtual appliances that support GENEVE
+- ELB -(TCP, SSL/TLS, HTTP, HTTPS (Layer 4 and 7) Basic load balancing for EC2 👈🏿 This is what we're going to work with!
 
-![LB Diagram](images/elb_network_diagram.png)
+[Here's the Docs on their differences in more detail](https://aws.amazon.com/elasticloadbalancing/features/#Product_comparisons)
+
+![LB Diagram](2022-images/3-elb-network-diagram.png)
```
Comment (bridgetbarnes) [inline part-2/2.2_setting_up_your_load_balancer.md:12]: I could be wrong, but I'm not sure this one is correct as it's not loading in the Github preview

Comment (bridgetbarnes) [reply part-2/2.2_setting_up_your_load_balancer.md:12]: nevermind, i didnt realise rich diff was turned on file by file

File: part-2/2.3_deploying_webservers.md [hunk 2]
```diff
@@ -20,18 +20,22 @@ These two instances will be our two web servers
 3. Click *Next: Configure Instance Details*
```
Comment (bridgetbarnes) [inline part-2/2.3_deploying_webservers.md:20]: On line 12 there's an example of the old diagram

**Review by bridgetbarnes — APPROVED** [review]
This is fantastic Franca!

File: part-2/2.2_setting_up_your_load_balancer.md (hunk 1 above)
Comment (francalovescakes) [reply part-2/2.2_setting_up_your_load_balancer.md:12]: that one is just a link :)

File: part-2/2.3_deploying_webservers.md (hunk 2 above)
Comment (francalovescakes) [reply part-2/2.3_deploying_webservers.md:20]: Updated!

**Comment by francalovescakes** [comment]
Yay

**Pull request merged**